javascript/javascript2/week1/currencyConverter.js

```diff
@@ -0,0 +1,82 @@
+// currency-converter-script.js
+
+document.addEventListener("DOMContentLoaded", () => {
+  const exRates = {
+    timestamp: Date.now(),
+    base: "USD",
+    date: new Date().toISOString().split("T")[0],
+    exRates: {
+      USD: 1.23396,
+      GBP: 0.882047,
+      EUR: 0.923987,
+      CAD: 1.36325,
+    },
+  };
+
+  const addNewRateform = document.getElementById("addNewRateform");
+  const convertCurrencyForm = document.getElementById("convertCurrencyForm");
+  const updateRateForm = document.getElementById("updateRateForm");
+  const addRateResult = document.getElementById("addRateResult");
+  const convertCurrencyResult = document.getElementById(
+    "convertCurrencyResult"
+  );
+  const updateRateResult = document.getElementById("updateRateResult");
+
+  addNewRateform.addEventListener("submit", (event) => {
+    event.preventDefault();
+    const baseCurrency = document
+      .getElementById("base-currency")
+      .value.toUpperCase();
```

Comment:
Great use of `toUpperCase()`. You also need to `trim` the value to avoid a situation like this (with two DKK rates):
<img width="162" alt="image" src="https://github.com/Rajesh-Bhatt/hyf-homework/assets/5467758/4f39d7f5-badd-46d9-a47c-136c1edfb2a7">

Reply:
trim() function implemented.